generator/stats.go

```diff
@@ -50,9 +47,6 @@ Most common action prefixes:
 {{- range .Prefixes }}
 | {{ $.Backtick }}{{ .Prefix }}{{ $.Backtick }} | {{ .Count }} |
 {{- end }}
-
-Thank you to [alanakirby/aktion](https://github.com/alanakirby/aktion) for originally 
```

Comment:
oi! y u no

Reply:
Because it's in README.md not stats.md

Reply:
https://github.com/glassechidna/trackiam/pull/17/files#diff-04c6e90faac2675aa89e2176d2eec7d8R13